packages/microsoft_kiota_http/lib/src/http_client_request_adapter.dart

```diff
@@ -2,14 +2,16 @@ part of '../microsoft_kiota_http.dart';
 
 class HttpClientRequestAdapter implements RequestAdapter {
   HttpClientRequestAdapter({
-    required http.Client client,
     required AuthenticationProvider authProvider,
-    required ParseNodeFactory pNodeFactory,
-    required SerializationWriterFactory sWriterFactory,
-  })  : _client = client,
+    ParseNodeFactory? pNodeFactory,
+    SerializationWriterFactory? sWriterFactory,
+    http.Client? client,
+  })  : _client = client ?? KiotaClientFactory.createClient(),
```

Comment:
nit: the `client` param is the last in the list; imho it should also be the last of these assignemnts

Reply:
thanks! fixed.